internal/publish/config.go

```diff
@@ -47,6 +47,8 @@ type Config struct {
 	Verification          verification.Config
 	ObservabilityExporter observability.Config
 
+	RevisionTokenConfig
```

Comment:
```suggestion
	RevisionToken RevisionTokenConfig
```

Reply:
We also generally don't put `Config` at the end since it's part of the `Config` struct.

Reply:
But here there are two Configs in this package. I could split it out elsewhere?